en/ui-system/components/editor/videoplayer.md

```diff
@@ -143,12 +143,11 @@ You can display a UI upon a video in two steps:
 
 2. Check the **stayOnBottom** property on the **VideoPlayer** in the **Properties** panel.
 
-**Note:**
-- This feature is only supported on Web.
-- The specific effects are not guaranteed to be consistent, depending on whether each browser supports or restricts.
-- After the **stayOnBottom** is enabled, the `clicked` event in `VideoPlayerEvent` cannot be listened normally.
+> **Note**: this feature is only supported on Web.
+> **Note**: the specific effects are not guaranteed to be consistent, depending on whether each browser supports or restricts.
+> **Note**: after the **stayOnBottom** is enabled, the `clicked` event in `VideoPlayerEvent` cannot be listened normally.
```

Comment:
This writing way need to add a newline character (<br>) after each point, otherwise you'll end up with the effect, which is a bit crowded:

![image](https://user-images.githubusercontent.com/19702899/100998031-fe6eb700-3595-11eb-88ef-ba93808b4ec8.png)

There are 3 writing styles, which one do you think is better? In the case of multiple notes. 

![image](https://user-images.githubusercontent.com/19702899/100998104-17776800-3596-11eb-91f8-44ece2acd6dc.png)

Reply:
We decided via WeChat that #2 was best. Thank you.